dz_s1/zadacha_7_from_seminar.py

```diff
@@ -0,0 +1,52 @@
+# Пользователь вводит число от 1 до 999. Используя операции с числами сообщите, что введено:
+# цифра, двузначное число или трехзначное число
+# Для цифры верните ее квадрат, например 5 - 25
+# Для двузначного числа - произведение цифр, например 30 - 0
+# Для трехзначного числа его трехначное отображение, например 520 - 25
+# Если число не из диапазона, запросите новое число
+# Откажитесь от магических числе
+# В коде должны быть один input и один print
+
+
+def get_number():
+    number_user = int(input('Введите число от 1 до 999: '))
+    if number_user > 999 or number_user < 0:
+        return get_number()
+    else:
+        return number_user
+
+
+number = get_number()
+print(number)
+number_2 = 0
+
+
+def find_count(number):
+    count = 0
+    while number > 0:
+        number = (number / 10) - (number % 10 / 10)
```

Comment:
Мне кажется, что есть подход по проще
```suggestion
    while number:
        number //= 10
```